server/controllers/Profile.js

```diff
@@ -23,4 +23,44 @@ const getProfile = async (req, res) => {
     }
 }
 
-module.exports = {getProfile};
\ No newline at end of file
+const addAllergy = async (req, res) => {
+    try {
+        const {id, UserId, ingredient} = req.body;
```

Comment:
Why do you need the profile id, if each profile has a userId shouldnt you just need the userId and the ingreident